numaprom/udf/postprocess.py

```diff
@@ -19,14 +20,15 @@
 _LOGGER = get_logger(__name__)
 
 HOST = os.getenv("REDIS_HOST")
-PORT = os.getenv("REDIS_PORT")
+PORT = int(os.getenv("REDIS_PORT", 6379))
 AUTH = os.getenv("REDIS_AUTH")
+MASTERNAME = os.getenv("REDIS_MASTERNAME")
```

Comment:
Shall we use separate env variables for sentinel host and port?

Reply:
Makes sense if we plan to use both the clients at the same time for some reason. WDYT?

Reply:
Based on the existence of env variables, to support different usecases.


Here is the approach in golang client implementation.

```golang
func NewUniversalClient(opts *UniversalOptions) UniversalClient {
	if opts.MasterName != "" {
		return NewFailoverClient(opts.Failover())
	} else if len(opts.Addrs) > 1 {
		return NewClusterClient(opts.Cluster())
	}
	return NewClient(opts.Simple())
}
```

https://github.com/redis/go-redis/blob/master/universal.go#L220-L227